pkg/smokescreen/smokescreen.go

```diff
@@ -256,7 +256,7 @@ func dialContext(ctx context.Context, network, addr string) (net.Conn, error) {
 	connTime := time.Since(start)
 
 	fields := logrus.Fields{
-		"conn_establish_time": connTime.String(),
+		"conn_establish_time_ms": connTime.Milliseconds(),
```

Comment:
mm, [it looks like this](https://pkg.go.dev/time#Duration.Milliseconds) will round everything to the nearest millisecond which means anything less than 1 millisecond will be logged as 0ms, which may be fine, if we don't care about that fine grain of timing on sub 1 millisecond requests.